src/index.js

```diff
@@ -1,7 +1,48 @@
+const client = require('prom-client');
 const config = require('config');
 const fetchStatuCakeData = require('./statuscake');
 const server = require('./server');
+const Storage = require('./statuscakedata');
 
-this.interval = setInterval(fetchStatuCakeData, config.statuscake.interval);
+const regex = /\s?([-])\s?/;
+
+const getTagsFromTitle = Title => Title.split(regex);
+
+const g = new client.Gauge({
+  name: 'metric_loadtime',
+  help: 'metric_help',
```

Comment:
What's the purpose of this attribute? 🙄

Reply:
Help attribute provides information about this metric. I will change for something more useful.